zeebe/snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/FileBasedSnapshotStoreImpl.java

```diff
@@ -0,0 +1,684 @@
+/*
```

Comment:
This is mainly a copy of the previous `FileBasedSnapshotStore`. The only changes are to replace the usage of actor.